azure-pipelines.yml

```diff
@@ -32,7 +32,7 @@ jobs:
       # It runs tests requiring pandas and PyAMG.
       pylatest_pip_openblas_pandas:
         DISTRIB: 'conda-pip-latest'
-        PYTHON_VERSION: '*'
+        PYTHON_VERSION: '3.7.4'  # FIXME: pinned until wheels are available
```

Comment:
why don't we use 3.7.5?

Reply:
I just checked `pylatest_conda` and it was this version but anyway it does not matter because we are going to unpinned it whenever wheels will be available on pypi.

Reply:
Let's put `PYTHON_VERSION='3.7'` :)